### ISSUE TITLE
How do you stop a GetContainerLogs stream via Go varlink interface?

### ISSUE THREAD

**Issue by johanbrandhorst** [containers/libpod#4077]
I've been playing around with the Go varlink API client for automating test container spin up and tear down. When using the API, I am able to Create, Start, Inspect and Get logs from the running container no problems. However, when I want to stop the container, sometimes the periodic message that comes in from the `GetContainerLogs` call will confuse the `StopContainer` call, since they are writing on the same stream (no multiplexing as far as I can see).

How is the user supposed to guard from this? The only reliable workaround I've found is to use `GetContainerLogs` only after stopping the container, which is not ideal since it's nice to have them in real time while interacting with the container.

Help appreciated.

**Comment by haraldh** [comment]
You can't make 2 calls on the same varlink connection. Just open another connection.

**Comment by haraldh** [comment]
Especially with connections with `varlink.More`
